src/citrine/resources/data_concepts.py

```diff
@@ -20,18 +20,59 @@
 
 
 class DataConcepts(PolymorphicSerializable['DataConcepts']):
-    """An abstract data concepts object. Must be extended along with Resource."""
+    """
```

Comment:
These aren't really part of the API, since DataConcepts is basically a metaclass (right?).  What's best practice for documenting them, specifically for separating out the docs that citrine-python user needs from those that a citrine-python developer needs (cc: @ahl389 )

Reply:
Good question. Many of the methods in that class definitely do not need to be documented for the user. For others do (such as filter_by_tags).

Reply:
I would argue that if documentation is comprised of "what it does" and "how to use it", both of those things are relevant to both a user and a developer, in slightly different proportions, and so even methods not exposed to a user should be duly documented (recognizing that time constraints are a real thing). 

Something like the `internal method` flag thats in this PR in the `build()` method is a great place to start (especially as stop gap if you want to skip full documentation of something), and a similar "for the user" flag would be good for any relevant code.

Reply:
I just checked @maxhutch and methods that begin with an underscore are not included in the sphinx docs. That means the only internal method users will see is build(), and that is already flagged.